tests/unit/test_downloader.py

```diff
@@ -28,7 +28,8 @@
 from yledl.backends import BaseDownloader, FailingBackend
 from yledl.downloader import YleDlDownloader
 from yledl.errors import TransientDownloadError
-from yledl.extractors import Clip, FailedClip, StreamFlavor
+from yledl.data_extractors import Clip
```

Comment:
Import error here, tests fail to run. Should be `from yledl.clip`  (if you rename clip_extracts.py -> clip.py)